src/v2/view-builder/views/password/ExpiredAuthenticatorPasswordView.js

```diff
@@ -0,0 +1,22 @@
+import { loc } from 'okta';
+import EnrollAuthenticatorPasswordView from './EnrollAuthenticatorPasswordView';
+
+const Body = EnrollAuthenticatorPasswordView.prototype.Body.extend({
```

Comment:
Curious, why are we extending this way?

Reply:
Discussed this with @haishengwu-okta - here's what we thought - 

There are at least 3 forms - admin init password reset (TBD), password will expire soon (TBD) and password expired (this PR) that have very similar structure. `EnrollAuthenticatorPasswordView` can be extended with minimal changes to achieve these.

Reply:
That makes sense! Thanks!